front-end/index.html

```diff
@@ -25,6 +25,8 @@
         <script src="https://cdn.jsdelivr.net/npm/gpu.js@2.0.0-rc.19/dist/gpu-browser.js"></script>
         <script src="https://cdn.jsdelivr.net/npm/jquery-validation@1.17.0/dist/jquery.validate.min.js"></script>
         <script src="https://d3js.org/d3.v5.min.js"></script>
+        <!-- <script src="https://unpkg.com/d3-simple-slider"></script> -->
```

Comment:
can we load this slider through a CDN?

Reply:
Actually, the slider is no longer needed so i can just delete this line!